tests/integration_tests/test_udfs.py

```diff
@@ -150,8 +126,9 @@
 #                         $input_table_name.col3
 #
 #                 FROM
-#                     $input_table_name
+#
+#                         $input_table_name
+#
 #             )
 #         )
-# );"""
-# )
+# );""")
```

Comment:
what are you gonna do with these...??

Reply:
These will be fixed from me on another PR, if you mean the commented out tests.